torch/nn/quantized/modules/conv.py

```diff
@@ -236,22 +256,10 @@ def from_float(cls, mod):
             mod = mod[0]
         else:
             activation_post_process = mod.activation_post_process
-        weight_post_process = mod.qconfig.weight()
-        weight_post_process(mod.weight)
-        act_scale, act_zp = activation_post_process.calculate_qparams()
-        assert weight_post_process.dtype == torch.qint8, \
-            'Weight observer must have a dtype of qint8'
-        qweight = _quantize_weight(mod.weight.float(), weight_post_process)
-        qconv = cls(mod.in_channels, mod.out_channels, mod.kernel_size,
-                    mod.stride, mod.padding, mod.dilation, mod.groups,
-                    mod.bias is not None, mod.padding_mode)
-        qconv.set_weight_bias(qweight, mod.bias)
-        qconv.scale = float(act_scale)
-        qconv.zero_point = int(act_zp)
-
-        return qconv
-
+        
```

Comment:
got some warnings in internal tools saying blank lines contains white space, could you remove the whitespace? (https://www.flake8rules.com/rules/W293.html)

Reply:
Yes. I will do that. Do I submit a new pull request after that?